resources/popup.ts

```diff
@@ -1,24 +1,27 @@
 "use strict";
 
+import colors from "./colors";
+
 export default function(
   streetname: string,
   wikidata?: string | null,
   name?: string | null,
   birth?: number | null,
   death?: number | null,
   description?: string | null,
+  gender?: string | null,
   wikipedia?: { lang: string; url: string } | null
 ): string {
   let html = "";
-
   if (
     typeof wikidata !== "undefined" &&
     wikidata !== null &&
     // to-do: Find out why wikidata is a "null" string (instead of null)
     wikidata !== "null"
   ) {
+    const highlightColor = gender ? colors[gender.toLowerCase()] : "#fff";
     html += '<div class="popup-wikidata">';
-    html += `<div class="popup-name">${name}</div>`;
+    html += `<h2 class="popup-name highlight-low" style="background:linear-gradient(180deg, rgba(255, 255, 255, 0) 65%, ${highlightColor} 65%);">${name}</h2>`;
```

Comment:
To be honest, I don't really like this highlight.
Good idea to "highlight" the gender but let's find another way !

Reply:
Do you dislike the style or the way it is applied?